aiida_vasp/calcs/immigrant.py

```diff
@@ -50,15 +49,29 @@ class VaspImmigrant(VaspCalculation):
 
        code = Code.get_from_string('vasp544mpi@local')
        folder = '/home/myself/vasp-calc-dir'
+       settings = {'parser_settings': {'add_energies': True,
+                                       'add_forces': True,
+                                       'electronic_step_energies': True}}
        VaspImmigrant = CalculationFactory('vasp.immigrant')
-       builder = VaspImmigrant.get_builder_from_folder(code, folder)
+       builder = VaspImmigrant.get_builder_from_folder(code,
+                                                       folder,
+                                                       settings=settings)
        submit(builder)
 
-    Instead of ``builder``, input dict is obtained similarly as
+    Instead of ``builder``, inputs dict is obtained similarly as
 
     ::
 
-       inputs = VaspImmigrant.get_inputs_from_folder(code, folder)
+       code = Code.get_from_string('vasp544mpi@local')
```

Comment:
This should be set in the calling script. And if we make an example we should follow the tutorial. I believe the entry now is `vasp@mycluster`.

Reply:
> This should be set in the calling script.

I don't know how to do it because this class is inherited from `VaspCalculation`. But we will not recommend users to use this `VaspImmigrant` calculation, but use `VaspImmigrantWorkChain` where it is designed as you wrote.

> And if we make an example we should follow the tutorial. I believe the entry now is vasp@mycluster.

OK. I see.
Immigrant is quite different. So I needed to write some document to be found by you. Finally the proper documentation has to be written in sphinx.

Reply:
Immigrant may not work for remote path, so `vasp@local` is better than `vasp@mycluster`.

Reply:
Aha, good point.